src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -141,10 +142,10 @@
  *
  * import java.io.Serializable;
  *
- * class Foo {
- *   public static final int FOO_CONST = 1;
+ * class FirstClass {
+ *   public static final int firstClassVariable = 1;
  *
- *   public void foo() {}
+ *   public void classMethod() {}
  * }
  * </pre>
  * <p>
```

Comment:
below this block , there is 2 Check configs.
please add java code example under each of them.

Reply:
Yes, in progress.

Reply:
New config example has been added repectively. Image has been shared below for the same:

![image](https://user-images.githubusercontent.com/90137881/194709293-d995ef29-0475-4203-ac7f-546dabb35d81.png)